Gemfile.lock

```diff
@@ -191,7 +191,7 @@ GEM
       railties (>= 4.2.0, < 6.0)
     rollbar (2.15.5)
       multi_json
-    rspec-core (3.8.1)
+    rspec-core (3.8.2)
```

Comment:
This is a dependent gem. I think it isn't necessary to upgrade it if it's not a security issue.
We will upgrade it if the main gem will upgrade it.

Reply:
If there are any incompatibilities, the constraints by the main gem won't allow updating, beside that I think it's better to do. Also if you run `bundle update` only for 1 gem, dependencies are not always updated.

Reply:
We couldn't be sure that the main gem would put the constraint if that is not compatible. 
It can happen when the main gem is not a fast maintained gem and we upgrade it's dependencies before it put the constraint.